tests/adapters/base.py

```diff
@@ -30,55 +70,167 @@ def test_initialize_model_input(self, test_adapter):
         """
         Test that initalizing model input does as intended.
         """
+        set_values = [0.9, 1.1]
+        test_adapter._output.get_writable_timeseries_view(
+            ("Emissions", "CO2"),
+            ("World",),
+            "GtC/yr",
+            [-10, 0, 10],
+            ParameterType.POINT_TIMESERIES,
+        ).set(set_values)
+
         assert not test_adapter._initialized
-        # TODO test for missing but mandatory parameter
-        test_adapter.initialize_model_input()
+        # start and end time should be set here as they're part of setting the
+        # timeseries views of emissions/concentrations/other drivers
+        test_adapter.initialize_model_input(start_time, end_time)
         assert test_adapter._initialized
+        # TODO test for start time and end time being passed properly
+        assert test_adapter._output.get_writable_timeseries_view(
+            ("Emissions", "CO2"),
+            ("World",),
+            "GtC/yr",
+            [-10, 0, 10],
+            ParameterType.POINT_TIMESERIES,
+        ).get() == set_values
```

Comment:
I don't think we need this test, it doesn't test anything about an adapter...

Reply:
If we have the convention that adapters should put input timeseries straight into `_output` then it tests that the adapter actually does that. Maybe that's not what we want though?

Reply:
@swillner thoughts here?